### PR TITLE
Changes in footer

### PR DESCRIPTION
**Pull request by mayurkapadia35**
--- moved footer from App.vue to each file separately
--- Add a trending post and daily-newsletter component in author page.
--- Fix the issue of baseImageUrl in authorSubmission page.
--- Bootstrap the file with v-app.

### CODE REVIEW

**Comment by ldeveloper7** [comment]
Give Proper Title & Description to Every PR

File: src/components/home/home.html [hunk 1]
```diff
@@ -154,8 +154,8 @@ <h2 class="font-weight-bold display-1">News and Features</h2>
             <v-btn block text-xs-center v-on:click="articleLoadMore(10)">Load More Stories</v-btn>
           </v-flex>
           <v-flex xs3 order-md3 order-xs3 color="red darken-4" text-xs-center>
-            <dailynewsletter/>
             <trendingpost/>
+            <dailynewsletter/>
```
Comment (ldeveloper7) [inline src/components/home/home.html]: remove this change...
make **dailynewsletter** first, and then **trendingpost**

Comment (mayurkapadia35) [reply src/components/home/home.html]: Solve that issue.

**Comment by mayurkapadia35** [comment]
Solve all the issues that are mentioned in PR.

**Comment by ldeveloper7** [comment]
Add **dailynewsletter** and **trendingpost** componant in author page

File: src/components/daily-newsletter/daily-newsletter.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <div>
+  <v-app>
```
[the remark this answers is not in the document]
Comment (mayurkapadia35) [reply src/components/daily-newsletter/daily-newsletter.html]: :+1:

File: src/components/daily-newsletter/daily-newsletter.html [hunk 3]
```diff
@@ -21,4 +22,5 @@
       </v-card-text>
     </v-form>
   </v-card>
+  </v-app>
```
[the remark this answers is not in the document]
Comment (mayurkapadia35) [reply src/components/daily-newsletter/daily-newsletter.html]: :+1:

File: src/components/home/home.html (hunk 1 above)
Comment (mayurkapadia35) [reply src/components/home/home.html]: :+1:

File: src/components/trending-post/trending-post.html [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <div>
+  <v-app>
```
[the remark this answers is not in the document]
Comment (mayurkapadia35) [reply src/components/trending-post/trending-post.html]: :+1:

File: src/components/trending-post/trending-post.html [hunk 5]
```diff
@@ -18,4 +19,5 @@ <h2 class="text-center underline text-decoration-none white--text ma-1 pa-1 pb-1
       </v-flex>
     </v-layout>
   </v-card>
+  </v-app>
```
[the remark this answers is not in the document]
Comment (mayurkapadia35) [reply src/components/trending-post/trending-post.html]: :+1:

**Comment by ldeveloper7** [comment]
Change **About** Link, its wrong.

**Comment by ldeveloper7** [comment]
we know that this is footer branch, give appropriate title

**Pull request merged**